packages/connect-thegraph/src/parsers/apps.ts

```diff
@@ -1,10 +1,13 @@
-import { AppData } from '@aragon/connect-core'
```

Comment:
Following are the changes to the parsers. Remember, they're returning entities now, not just data objects.